src/sec_certs/sample/cve.py

```diff
@@ -48,11 +48,12 @@ def from_nist_dict(cls, dct: dict[str, Any]) -> CVE.Impact:
 
     cve_id: str
     vulnerable_cpes: list[CPE]
+    vulnerable_cpe_configurations: list[CPEConfiguration]
```

Comment:
Just noticed this. Is there a reason that we have this as a list? Could be set as well, right?

Reply:
There is no any specific reason for storing `CPEConfiguration` as a list. I just wanted to keep the same data structure as the `CPE` records have.

Reply:
Sure, that makes sense. Maybe  there's a reason why CPEs are a list, but I cannot recall any. Could you pls try to refactor both to sets and see what happens?

Reply:
@adamjanovsky Refactoring seems to be okay - at least the pipeline of downloading CVEs, processing them and building the `CVEDataset` is passing. Due to the usage of `CVEs` in `pandas_columns` I will also investigate the Jupyter notebooks to check if the change does not break the analysis code.